index.js

```diff
@@ -47,12 +36,13 @@ app.post('/', upload.single('thumb'), function (req, res, next) {
     const playEmoji = req.query.playEmoji || process.env.PLAY_EMOJI;
     const pauseEmoji = req.query.pauseEmoji || process.env.PAUSE_EMOJI;
```

Comment:
Only because I'm paranoid about errors and _think_ both cases will return `undefined` vs an empty string, can we add ` || ''` to the emoji selections so they are never `undefined`?